### PR TITLE
innit

### PR DESCRIPTION
**Pull request by denisdugar**

### CODE REVIEW

File: sort/sort_test.go [hunk 1]
```diff
@@ -1,87 +1,37 @@
...
-	if !(reflect.DeepEqual(arrSortMy, arrSort4)){
-		t.Errorf("got %q, want %q", arrSortMy, arrSort4)
-	}
-	if !(reflect.DeepEqual(arrSortGo, arrSort4)){
-		t.Errorf("got %q, want %q", arrSortGo, arrSort4)
-	}
-}
\ No newline at end of file
+var testCases = []struct {
+	actual   []int
```
Comment (Dimitriy14) [inline sort/sort_test.go:10]: Add a name field and some words about each testCase

File: sort/sort_test.go [hunk 2]
```diff
@@ -1,87 +1,37 @@
...
+	{actual: []int{1}, expected: []int{1}},
+	{actual: []int{}, expected: []int{}},
+	{actual: []int{1, 2, 3, 4, 5}, expected: []int{1, 2, 3, 4, 5}},
+}
+
+func TestFlagParser(t *testing.T) {
+	var a []int
+	var b []int
+	for _, tt := range testCases {
+		t.Run("test", func(t *testing.T) {
```
Comment (Dimitriy14) [inline sort/sort_test.go:24]: Could you please use tt.name istead of hardcoded "test"

File: sort/sort_test.go [hunk 3]
```diff
@@ -1,87 +1,37 @@
...
+	{actual: []int{5, 7, 3, 8, 1, 9, 0, 2, 5, 10}, expected: []int{0, 1, 2, 3, 5, 5, 7, 8, 9, 10}},
+	{actual: []int{1, 1, 1, 1, 1, 1, 1, 1}, expected: []int{1, 1, 1, 1, 1, 1, 1, 1}},
+	{actual: []int{1}, expected: []int{1}},
+	{actual: []int{}, expected: []int{}},
+	{actual: []int{1, 2, 3, 4, 5}, expected: []int{1, 2, 3, 4, 5}},
+}
+
+func TestFlagParser(t *testing.T) {
+	var a []int
+	var b []int
```
Comment (Dimitriy14) [inline sort/sort_test.go:22]: Is there any point to use this slices? Can we just use actual slices directly?

File: sort/sort_test.go [hunk 4]
```diff
@@ -1,87 +1,37 @@
...
+	for _, tt := range testCases {
+		t.Run("test", func(t *testing.T) {
+			a = tt.actual
+			SortT(a)
+			b = tt.actual
+			sort.Ints(b)
```
Comment (Dimitriy14) [inline sort/sort_test.go:28]: Please create another test function for this sort

File: sort/sort_test.go [hunk 5]
```diff
@@ -1,87 +1,37 @@
...
\ No newline at end of file
+var testCases = []struct {
+	actual   []int
+	expected []int
+}{
+	{actual: []int{5, 7, 3, 8, 1, 9, 0, 2, 5, 10}, expected: []int{0, 1, 2, 3, 5, 5, 7, 8, 9, 10}},
+	{actual: []int{1, 1, 1, 1, 1, 1, 1, 1}, expected: []int{1, 1, 1, 1, 1, 1, 1, 1}},
+	{actual: []int{1}, expected: []int{1}},
+	{actual: []int{}, expected: []int{}},
+	{actual: []int{1, 2, 3, 4, 5}, expected: []int{1, 2, 3, 4, 5}},
```
Comment (Dimitriy14) [inline sort/sort_test.go:17]: Please add one more test case with nil slices

**Review by Dimitriy14 — CHANGES_REQUESTED** [review]

**Review by Dimitriy14 — APPROVED** [review]
